yt/frontends/openPMD/data_structures.py

```diff
@@ -30,6 +30,19 @@
 import os
 
```

Comment:
offline discussion with @Rahmspinat:
Instead  of `mylog.debug()` we need to use `mylog.warning()` to see the warning on the ipython screen.